### PR TITLE
Chart adjust title; tooltip: add space, remove percentages

### PR DESCRIPTION
**Pull request by Skywalker-11**

### CODE REVIEW

**Comment by stefaweb** [comment]
Possible to have % when <3%?

**Comment by Skywalker-11** [comment]
done

File: mailscanner/lib/chartConfig.js [hunk 1]
```diff
@@ -102,7 +103,11 @@ var myChart = new Chart(ctx, {
             total += allData[i];
           }
           var tooltipPercentage = Math.round((itemData / total) * 100);
-          return tooltipLabel + ": " + chartFormattedData[tooltipItem.index] + " (" + tooltipPercentage + "%)";
+          var tooltipOutput = " "+ tooltipLabel + ": " + chartFormattedData[tooltipItem.index];
```
Comment (stefaweb) [inline mailscanner/lib/chartConfig.js]: Need to use colon99 in `": "`.

Comment (Skywalker-11) [inline mailscanner/lib/chartConfig.js]: little hacky but done

File: mailscanner/lib/chartConfig.js [hunk 2]
```diff
@@ -103,7 +103,7 @@ var myChart = new Chart(ctx, {
             total += allData[i];
           }
           var tooltipPercentage = Math.round((itemData / total) * 100);
-          var tooltipOutput = " "+ tooltipLabel + ": " + chartFormattedData[tooltipItem.index];
+          var tooltipOutput = " "+ tooltipLabel + COLON + chartFormattedData[tooltipItem.index];
```
Comment (stefaweb) [inline mailscanner/lib/chartConfig.js]: Space after colon should be outside COLON

Comment (Skywalker-11) [inline mailscanner/lib/chartConfig.js]: done

**Pull request merged**